### PR TITLE
Website

### PR DESCRIPTION
**Pull request by trace2798**
Added the website. Working on docs and example layout

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "website",
```
Comment (iamvishnusankar) [inline website/package.json:2]: Set `name: "docs"`

**Review by iamvishnusankar — CHANGES_REQUESTED** [review]
@trace2798 Thanks for this PR 🙏 

Here're some suggestion: 
- Rename `website` folder to docs
- Edit the root `package.json` and add the following in `workspaces` entry:
 ```
 {
    "workspaces": {
        "packages": [
           "packages/*",
            "examples/*",
            "docs"
        ]
  }
  ```

File: docs/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "next-themes": "^0.2.1",
+    "postcss": "8.4.23",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "tailwind-merge": "^1.12.0",
+    "tailwindcss": "3.3.2",
+    "typescript": "5.0.4"
+  },
+  
+   "workspaces": {
```
Comment (iamvishnusankar) [inline docs/package.json:31]: Not here, add it on root `package.json`. This file https://github.com/iamvishnusankar/next-sitemap/blob/master/package.json

**Review by iamvishnusankar — CHANGES_REQUESTED** [review]

File: docs/package.json (hunk 2 above)
Comment (trace2798) [reply docs/package.json:31]: My bad got it now tho

File: docs/yarn.lock
Comment (iamvishnusankar) [inline docs/yarn.lock:1]: @trace2798 Remove this file in favour of root yarn.lock. Run install commands on root directory

**Review by iamvishnusankar — CHANGES_REQUESTED** [review]

**Review by iamvishnusankar — APPROVED** [review]
@trace2798 Thanks for this PR! Approved 🙏

**Pull request merged**